tests/test_errata.py

```diff
@@ -79,7 +79,10 @@ def __init__(self, errata_rest_get, errata_http_get=None):
             "product": {
                 "id": 89,
                 "short_name": "product",
-            }
+            },
+            "people": {
+                "reporter": "botas/pnt-devops-rad-jenkins.rhev-ci-vms.eng.rdu2.redhat.com@REDHAT.COM"
```

Comment:
We should avoid having internal info like this in code.

Reply:
Ok I will change it